html-block-and-inline/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <style>
+
+  </style>
+</head>
+<body>
+  <div>
+    <h1> Blue Americana Snuggie &reg; Blanket</h1>
+    <h2> by SnuggieStore</h2>
+    <p>
+      Price: <strike>$19.99</strike> $14.99 <br/>
+      <br />
+      Part of our new Americana collection, theis <i>beautiful</i> blue is already a customer favorite!<br />
```

Comment:
Although `<i>` works here, it is best to use `<em>` for italics. The `<em>` tag is recognized by screen reader whereas `<i>` tags are not.